src/profiler/profiler.c

```diff
@@ -56,6 +56,8 @@ void xdebug_profiler_mshutdown(void)
 void xdebug_profiler_rinit(void)
 {
 	XG_PROF(profile_file)  = NULL;
+    XG_PROF(profile_file_buffer_len) = 0;
+	XG_PROF(profile_file_buffer) = NULL;
```

Comment:
I don't want to add another home grown algorithm for string buffering. We have `xdebug_str`, which now also has a `xdebug_str_fmt` version since a week or two. That would also prevent us from having to make an assumption that 128kb for a line/function is big enough, which it might as well not have.
So this needs changing to a `xdebug_str*`.